pkg/network/dns/monitor_linux.go

```diff
@@ -56,6 +56,11 @@ func NewReverseDNS(cfg *config.Config) (ReverseDNS, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if err := p.Start(); err != nil {
```

Comment:
@brycekahle @p-lambert Why we never start the manager ?

Reply:
I got it, as we got the pkts on userspace .... so I will implement a classifier probe as we can't have both behavior at the same time